src/main/java/frc/robot/subsystems/Claw.java

```diff
@@ -67,7 +67,7 @@ public double getPosition () {
   @Override
   public void periodic() {
     //figure out how to get the shuffleboard to output the position of the encoder here
-    SmartDashboard.putNumber(sEncoderName, encoder.getPosition());
+    //SmartDashboard.putNumber(sEncoderName, encoder.getPosition());

```

Comment:
did we mean to remove this?

Reply:
yes, we added a different way to output them so that we can see both instead of just the left one